### DISCUSSION TITLE
https://cdn.jsdelivr.net/npm/@vue/compiler-sfc@3.4.27/dist/compiler-sfc.esm-browser.js cannot access, index demos cannot  show normally

### DISCUSSION THREAD

**Discussion by liang0-0** [bcakmakoglu/vue-flow#1660] [General]
![image](https://github.com/user-attachments/assets/2a392e67-e9c4-4565-8e53-79f48804a3dd)
![image](https://github.com/user-attachments/assets/c3c0c6ab-6806-44bb-8a71-99b347cc4a92)

now , I cannot view the examples in vueflow website, its very offen 
hope u can change another cdn link, or use some [liberary cdn fallback](https://github.com/PipecraftNet/jsdelivr-auto-fallback) 3ks,

**Comment by bcakmakoglu** [comment]
I have never experienced this issue ever so I’m assuming it has to be something on your end tbh.
